README.md

```diff
@@ -66,20 +77,116 @@ sentry::path: '/var/lib/sentry'
 sentry::version: '7.7.1'
 ```
 
+## Classes
```

Comment:
I don't know that there is a whole lot of value in enumerating the classes and defined types and their parameters.  In my opinion this will get out of date.  I like the automation section.